data/json/mapgen/lab/lab_rooms.json

```diff
@@ -76,26 +84,19 @@
     "object": {
       "mapgensize": [ 7, 7 ],
       "rotation": [ 0, 3 ],
-      "rows": [
```

Comment:
Lets's skip linting files with small nested mapgen like this for now, it renders them unreadable.
I'll need to sort out a fix for this.

Reply:
No problem. I'll skip the mapgen ones until you give me an okay.

Reply:
Mind if I close the pull request then?

Reply:
Go ahead if it makes things easier.
And feel free to e.g. reorder entries in the blacklist if it makes it easier to keep track of your progress.

Reply:
Will do, thank you.